tests/bsim/bluetooth/samples/compile.sh

```diff
@@ -17,25 +17,22 @@ app=samples/bluetooth/peripheral_hr \
   compile
 app=tests/bsim/bluetooth/samples/central_hr_peripheral_hr \
   sysbuild=1 \
-  extra_conf_file=${ZEPHYR_BASE}/samples/bluetooth/central_hr/prj.conf \
```

Comment:
Please reuse the conf  file from samples, so that CI covers any future changes to samples.